app/src/main/res/layout/ether_transaction_bottom_sheet.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/ether_transaction_bottom_sheet_layout"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/bottom_sheet_big_height"
+    android:background="@drawable/rounded_dialog"
+    android:orientation="vertical"
+    android:paddingTop="32dp"
+    android:paddingHorizontal="32dp"
+    android:paddingBottom="36dp"
+    android:clipToPadding="false"
+    android:clipChildren="false"
+    >
+
+  <TextView
+      android:id="@+id/ether_transaction_bottom_sheet_title"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="The transaction has started"
+      style="@style/BottomSheetEtherTransactionTitle"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      />
+  <TextView
+      android:id="@+id/ether_transaction_bottom_sheet_subtitle"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="11dp"
+      android:text="You can follow the process on etherscan"
+      style="@style/BottomSheetEtherTransactionSubtitle"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/ether_transaction_bottom_sheet_title"
+      app:layout_constraintStart_toStartOf="parent"
+      />
+  
+  <androidx.constraintlayout.widget.ConstraintLayout
+      android:id="@+id/ether_transaction_bottom_sheet_rectangle"
+      android:layout_width="match_parent"
+      android:layout_height="88dp"
+      android:layout_marginTop="24dp"
+      android:background="@drawable/rectangle_outline_darkblue_radius_8dp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/ether_transaction_bottom_sheet_subtitle"
+      android:paddingVertical="8dp"
+      android:paddingHorizontal="8dp">
+    <ImageView
+        android:id="@+id/ether_transaction_bottom_sheet_etherscan_logo"
+        android:layout_width="100dp"
+        android:layout_height="22dp"
+        android:background="@drawable/ic_etherscan_logo"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"/>
+
+    <TextView
+        android:id="@+id/ether_transaction_bottom_sheet_hash_string"
+        android:layout_width="168dp"
+        android:layout_height="32dp"
+        android:text="0x0bb0505b1fa7e4d106ae5b2fcc1f9844b606671"
```

Comment:
Generally we replace this with `tools:text="..."` after testing so that we can still visualize in the editor but so that there's no chance of letting display mocked data to the user (for whatever reason).